### PR TITLE
docs: added en version of show components

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
 added en version of show components
modified the index.rst to change the contents (include the show_table_status)

### CODE REVIEW

**Comment by michelle-qinqin** [comment]
下述PR对index.rst文件的修改需求合并到本PR完成
https://github.com/4paradigm/OpenMLDB/pull/2376

File: docs/en/reference/sql/ddl/SHOW_COMPONENTS.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+| Endpoint     | It shows the endpoint of the component by providing the IP and the port, which is the same as the `--endpoint` flag in configuration files.                                                       |
+| Role         | It indicates the role of the component, which is the same as the `--role` flag in configuration files. <br/> There are four types of role: `tablet`, `nameserver`, `taskmanager` and `apiserver`. |
+| Connect_time | It shows the connecting momment of the component by timestamp in milliseconds.                                                                                                                    |
+| Status       | It shows the status of the component. There are three kinds of status: `online`, `offline` and `NULL`.                                                                                            |
+| Ns_role      | It shows the role of the Namserver: `master` or `standby`.                                                                                                                                        |
```
Comment (aceforeverd) [inline docs/en/reference/sql/ddl/SHOW_COMPONENTS.md]: for other component role, `Ns_role` is NULL

File: docs/en/reference/sql/ddl/index.rst [hunk 2]
```diff
@@ -13,8 +13,10 @@ Data Definition Statement (DDL)
     DESC_STATEMENT
     CREATE_TABLE_STATEMENT
     DROP_TABLE_STATEMENT
+    SHOW_COMPONENTS
```
Comment (aceforeverd) [inline docs/en/reference/sql/ddl/index.rst:16]: fix the indent problem ?

Comment (michelle-qinqin) [reply docs/en/reference/sql/ddl/index.rst:16]: 我本地编译是正常的

**Review by lumianph — APPROVED** [review]

**Pull request merged**